crossbeam-skiplist/src/equivalent.rs

```diff
@@ -0,0 +1,49 @@
+use core::{borrow::Borrow, cmp::Ordering};
```

Comment:
```suggestion
// These traits are based on `equivalent` crate, but `K` and `Q` are flipped to avoid type inference issues:
// https://github.com/indexmap-rs/equivalent/issues/5

use core::{borrow::Borrow, cmp::Ordering};
```

Reply:
Add the doc, but GH does not show the changes.